docs/index.md

```diff
@@ -45,205 +56,161 @@ theme:
 
-Check out the scripts in our `examples` folder to see some of the functionality
-we're developing!
+# Community
 
-**napari** supports six main different layer types, `Image`, `Labels`, `Points`,
-`Vectors`, `Shapes`, and `Surface`, each corresponding to a different data type,
-visualization, and interactivity. You can add multiple layers of different types
-into the viewer and then start working with them, adjusting their properties.
+## Code of conduct
 
-All our layer types support n-dimensional data and the viewer provides the
-ability to quickly browse and visualize either 2D or 3D slices of the data.
+Napari has a [Code of Conduct](./community/code_of_conduct) that should be
+honored by everyone who participates in the `napari` community.
 
-**napari** also supports bidirectional communication between the viewer and the
-Python kernel, which is especially useful when launching from jupyter notebooks
-or when using our built-in console. Using the console allows you to
-interactively load and save data from the viewer and control all the features of
-the viewer programmatically.
+## Ask questions
 
-You can extend **napari** using custom shortcuts, key bindings, and mouse
-functions.
+Still have questions? We use the [image.sc
+forum](https://forum.image.sc/tag/napari) to answer usage questions about
+napari.
 
-## Tutorials
+## Report bugs
 
-For more details on how to use `napari` checkout our [tutorials](./tutorials/index).
-These are still a work in progress, but we'll be updating them regularly.
+If you encounter any issues while using napari, please 
 
-## Mission, values, and roadmap
+## Chat with us
 
-For more information about our plans for `napari` you can read our [mission and
-values statement](./community/mission_and_values.md), which includes more
-details on our vision for supporting a plugin ecosystem around napari.
+We chat on a platform called [Zulip](https://napari.zulipchat.com). It's a good
+place to chat informally about the project, or ask someone from the community
+for directions. If you're not sure about the best way to engage with the
+project or whether napari can be useful for you, drop by!
 
-## Contributing
+## Meet with us
 
-Contributions are encouraged! Please read our [contributing
-guide](./developers/contributing.md) to get started. Given that we're in an
-early stage, you may want to reach out on our [Github
-Issues](https://github.com/napari/napari/issues) before jumping in.
+Napari runs weekly or fortnightly meetings that anyone is welcome to attend,
+where we discuss project progress and development. Additionally, we have a
+number of working groups which meet with varying frequency about specific
+topics of napari development. To participate, see the [community
+calendar](community/meeting_schedule). (It might be a good idea to drop us a
+line on [Zulip](https://napari.zulipchat.com) so we know to expect you!)
 
-## Code of conduct
+[maybe embed calendar here]
```

Comment:
I like the idea of embedding the calendar on the homepage as I think a lot of people come to the website to look for meetings. I also like keeping it in the community section so we can have the direct link.